src/Build/Logging/BinaryLogger/BuildEventArgsFieldFlags.cs

```diff
@@ -22,6 +22,8 @@ internal enum BuildEventArgsFieldFlags
         LineNumber = 1 << 10,
         ColumnNumber = 1 << 11,
         EndLineNumber = 1 << 12,
-        EndColumnNumber = 1 << 13
```

Comment:
OUCH! That is totally my typo, from 2017. Well spotted, and how embarrassing. Can't believe I never noticed. Will fix.

Reply:
And yes, we can only have 32. One tweak we can do when we run out is change the enum base type to long, this will buy us 32 more. But I don't think we'll be adding much more to this.